cmd/libvirt-actuator/resources/init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# Update the qemu-kvm to a version that has -fw_cfg flag (2.4 at least)
+cat <<EOF |
+[centos]
+name=centos
+baseurl=http://mirror.centos.org/centos/7/virt/x86_64/kvm-common/
+gpgcheck=0
+enabled=1
+EOF
+ssh libvirtactuator 'cat > /etc/yum.repos.d/centos.repo'
+
+ssh libvirtactuator 'yum update qemu-kvm -y'
+
+# Create default storage volume
+export VIRSH="virsh -c qemu+ssh://root@libvirtactuator/system"
```

Comment:
This would be better as:

    export LIBVIRT_DEFAULT_URI=qemu+ssh://root@libvirtactuator/system

Then all `${VIRSH}` can be just `virsh`.